etl/steps/data/meadow/ggdc/2022-11-28/penn_world_table_national_accounts.meta.yml

```diff
@@ -1,11 +1,24 @@
 dataset:
```

Comment:
Same as with the other dataset: If the metadata in meadow and garden are identical, you don't need to have a metadata.yaml file both in meadow and in garden. You can just keep one of them (e.g. the one in meadow) and copy meadow metadata to garden.